wiki/Main_Page/es.md

```diff
@@ -62,7 +59,7 @@ Actividades: [Beatmapping](/wiki/Beatmapping) • [Técnicas para mapping](/wiki
 
 # Envío y clasificación de Beatmap
 
-La clasificación es el proceso de obtener una tabla de clasificación para un beatmap, lo que lo hace más popular y reconocido. Descubra lo que se necesita para tomar este camino y cómo aumentar sus posibilidades de éxito con las revisiones de su beatmap, así como los procedimientos de control de calidad y el proceso de clasificación.
+La clasificación es el proceso de obtener una tabla de clasificación para un beatmap, lo que lo hace más popular y reconocido. Descubra lo que se necesita para recorrer este camino y cómo aumentar sus posibilidades de éxito a través de las revisiones de su beatmap, así como los procedimientos de control de calidad y el proceso de clasificación.
 
 [Envío](/wiki/Submission) • [Modding](/wiki/Modding) • [Procedimiento de clasificación](/wiki/Beatmap_ranking_procedure) • [Mappers' Guild](/wiki/Community/Mappers_Guild) • [Proyecto Amado](/wiki/Community/Project_Loved)
```

Comment:
While we are here I would rename:
```suggestion
[Envío](/wiki/Submission) • [Modding](/wiki/Modding) • [Procedimiento de clasificación](/wiki/Beatmap_ranking_procedure) • [Mappers' Guild](/wiki/Community/Mappers_Guild) • [Proyect Loved](/wiki/Community/Project_Loved)
```

Given it is a "nombre propio" considering it is capitalized.